index.html

```diff
@@ -31,7 +31,167 @@ <h1>Charlie Giorgania</h1>
+          </button>
+        </div>
+
+        <div class="projImg2"></div>
+
+        <div class="container2">
+          <p class="projName">Project name goes here</p>
+          <ul class="stacks">
+            <li>HTML/CSS</li>
+            <li>Ruby on Rails</li>
+            <li>JavaScript</li>
+          </ul>
+
+          <button class="projView">
+            See this Project
+            <img class="arrow2" src="./images/arrow.png" alt="" />
+          </button>
+        </div>
+
+        <div class="projImg3"></div>
+
+        <div class="container3">
+          <p class="projName">Project name goes here</p>
+          <ul class="stacks">
+            <li>HTML/CSS</li>
+            <li>Ruby on Rails</li>
+            <li>JavaScript</li>
+          </ul>
+
+          <button class="projView">
+            See this Project
+            <img class="arrow" src="./images/arrow.png" alt="" />
+          </button>
+        </div>
+
+        <div class="projImg4"></div>
+
+        <div class="container4">
+          <p class="projName">Project name goes here</p>
+          <ul class="stacks">
+            <li>HTML/CSS</li>
+            <li>Ruby on Rails</li>
+            <li>JavaScript</li>
+          </ul>
+
+          <button class="projView">
+            See this Project
+            <img class="arrow" src="./images/arrow.png" alt="" />
+          </button>
+        </div>
+
+        <div class="projImg5"></div>
+
+        <div class="container5">
+          <p class="projName">Project name goes here</p>
+          <ul class="stacks">
+            <li>HTML/CSS</li>
+            <li>Ruby on Rails</li>
+            <li>JavaScript</li>
+          </ul>
+
+          <button class="projView">
+            See this Project
+            <img class="arrow" src="./images/arrow.png" alt="" />
+          </button>
+        </div>
+
+        <div class="projImg6"></div>
+
+        <div class="container6">
+          <p class="projName">Project name goes here</p>
+          <ul class="stacks">
+            <li>HTML/CSS</li>
+            <li>Ruby on Rails</li>
+            <li>JavaScript</li>
+          </ul>
+
+          <button class="projView">
+            See this Project
+            <img class="arrow" src="./images/arrow.png" alt="" />
+          </button>
+        </div>
+      </section>
+
+      <section>
+        <div class="aboutMe"><h2>About</h2></div>
+        <p class="aboutDescription">
+          Lorem ipsum dolor sit amet, consectetur <br />
+          adipiscing elit, sed do eiusmod tempor<br />
+          incididunt ut labore et dolore magna <br />
+          aliqua. Ut enim ad minim veniam,quis<br />
+          nostrud exercitation ullamco laboris nisi<br />
+          ut aliquip ea eiusmod
+        </p>
+
+        <div class="aboutButton">
+          <p>Get my CV</p>
+          <img class="aboutArrow1" src="./images/aboutarrow1.png" alt="" />
+          <img class="aboutArrow" src="./images/aboutarrow.png" alt="" />
+        </div>
```

Comment:
[OPTIONAL]
- [ ] The button (e.g., "Get my CV") should be a link `a` or `button` tag, and follow the button styling: Active, Hover and Pressed (see design)

![image](https://user-images.githubusercontent.com/29071984/177009992-4bad4b78-7df1-47ae-a400-44192ba63ebd.png)